### DISCUSSION TITLE
For Some Questions

### DISCUSSION THREAD

**Discussion by bajrangCoder** [TomSchimansky/CustomTkinter#818] [Q&A]
I have made a beautiful desktop app using `customtkinter` but I have used `tkinter.messagebox` and tkinter ttk treeview for showing table . But tkinter message box and ttk treeview decrease the beauty of my app . is there any way by which i can make custom message box and table view in custom tkinter .

Please help!

**Comment by Akascape** [comment]
@bajrangCoder Here is a table widget for customtkinter: https://github.com/Akascape/CTkTable
![Screenshot](https://user-images.githubusercontent.com/89206401/233420929-bf210cb3-5b5f-49b2-ba7a-f01d187e72cf.jpg)
and also a messagebox: https://github.com/Akascape/CTkMessagebox
![Screenshot](https://github.com/Akascape/CTkMessagebox/assets/89206401/6d92106e-27c7-4ccd-b16b-06a48d2b62f5)
Other treeview style: https://github.com/TomSchimansky/CustomTkinter/discussions/524#discussioncomment-3898307

**Reply by bajrangCoder** [reply]
Yes , I had already seen it.
By the way Thanks
